features/pom.xml

```diff
@@ -124,7 +123,7 @@
         <cxf.hazelcast.version>3.9.2</cxf.hazelcast.version>
         <cxf.hibernate.validator.version>5.3.6.Final</cxf.hibernate.validator.version>
         <cxf.httpcomponents.asyncclient.version>4.1.3</cxf.httpcomponents.asyncclient.version>
-        <cxf.httpcomponents.core.version>4.4.9</cxf.httpcomponents.core.version>
+        <cxf.httpcomponents.core.version>4.4.10</cxf.httpcomponents.core.version>
```

Comment:
Looks like we could also use https://github.com/codice/ddf-parent/blob/master/pom.xml#L52 here.